doc/classes/MultiplayerPeer.xml

```diff
@@ -97,49 +97,34 @@
 		</member>
 	</members>
 	<signals>
-		<signal name="connection_failed">
-			<description>
-				Emitted when a connection attempt fails.
-			</description>
-		</signal>
-		<signal name="connection_succeeded">
-			<description>
-				Emitted when a connection attempt succeeds.
-			</description>
-		</signal>
 		<signal name="peer_connected">
 			<param index="0" name="id" type="int" />
 			<description>
-				Emitted by the server when a client connects.
+				Emitted when a remote peer connects.
```

Comment:
So, with these changes, we should make the exact order clear. Is this signal emitted when a remote peer _has_ successfully connected? Or is it emitted when the peer is _trying_ to connect (and maybe needs to auth)?